__tests__/app.test.js

```diff
@@ -129,4 +129,103 @@ describe("/api/articles/:article_id", () => {
         });
     });
   });
+  describe("PATCH", () => {
+    test("Status:201, the body should contain a property of inc_votes and should have a number as a value", () => {
```

Comment:
I don't feel like this test is adding anything here, ideally this would be taken care of in error handling. All you are asserting is that the object written on line 134 is correct. This test could never fail.